src/app/shared/state/entry-wizard.service.ts

```diff
@@ -117,7 +117,7 @@ export class EntryWizardService {
     this.entryWizardStore.setLoading(true);
     const registryEnum = this.convertSourceControlStringToEnum(repository.gitRegistry);
     this.workflowsService
-      .deleteWorkflow(registryEnum, repository.organization, repository.repositoryName)
+      .deleteWorkflow(registryEnum as 'github.com' | 'bitbucket.org' | 'gitlab.com', repository.organization, repository.repositoryName)
```

Comment:
Curious, why did this change?

Reply:
This was a fix that's since been merged to develop, it now matches.